.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,11 +1,11 @@
 blank_issues_enabled: true
 contact_links:
   - name: Suggest a feature
-    url: https://portal.productboard.com/winappsdk/1-windows-app-sdk
+    url: https://github.com/microsoft/WindowsAppSDK/issues
```

Comment:
Giving a link to the issues page in this file is unhelpful because this file is displayed when a user is trying to create a new issue, and therefore has already been to that page. Furthermore, obstructing the user when they choose “Feature request” makes it appear that feature requests are unwelcome.